javaparser-testing/src/test/resources/com/github/javaparser/bulk_test_results/openjdk_test_results.txt

```diff
@@ -473,6 +473,10 @@ Lexical error at line 8, column 9.  Encountered: "#" (35), after : ""
 openjdk/langtools/test/tools/javac/rawDiags/Error.java
 (line 9,col 17) Parse error. Found <EOF>, expected "("
 
+openjdk/langtools/test/tools/javac/StoreClass.java
+(line 35,col 9) Illegal left hand side of an assignment.
```

Comment:
Why I'm doing that is so that I can see how a change in the grammar affects the parse results. If it wasn't checked in, I couldn't use the diff tools to see what the effect of a grammar or validation change is. It's invaluable to me :-)